dist/index.js

```diff
@@ -22106,7 +22121,7 @@ function run() {
                 yield uploadArtifact(scanResult);
             }
         }
-        catch (_a) {
+        catch (_b) {
```

Comment:
I dont see a new variable `_b` Not sure if this change is intentional.

Reply:
The files under `dist/` directory are auto-generated and the variable names get minified there. You can ignore these files.